x-pack/plugins/graph/public/_main.scss

```diff
@@ -21,6 +21,7 @@
  */
 
 .gphNoUserSelect {
+  padding-right: $euiSizeXS;
```

Comment:
I see this adds a bit of padding to the <p> node labels, but it's not clear what problem this addresses. Out of curiosity, was there a particular situation that led to this addition?

Reply:
I guess it was mistakenly leaved and can be removed in a follow up PR.